### PR TITLE
Fixed typo (iterative --> interactive)

### PR DESCRIPTION
**Pull request by teacherc**
I think "iterative mode" should be "interactive mode".

### CODE REVIEW

**Comment by teacherc** [comment]
I fixed a small typo in the homework and added my notes to the readme.

File: week_1_basics_n_setup/README.md [hunk 1]
```diff
@@ -100,6 +100,7 @@ If you have problems setting up the env, you can check this video:
 
 Did you take notes? You can share them here
 
+* [Notes from Candace Williams - 2023 Cohort](https://teacherc.github.io/data-engineering/2023/01/18/zoomcamp1.html)
```
Comment (alexeygrigorev) [inline week_1_basics_n_setup/README.md:103]: Can you please put your notes at the end?

Comment (teacherc) [reply week_1_basics_n_setup/README.md]: I just put my notes at the end, @alexeygrigorev.

**Comment by alexeygrigorev** [comment]
Thank you!

**Pull request merged**